query-engine/connector-test-kit-rs/query-engine-tests/src/utils/querying.rs

```diff
@@ -7,16 +7,32 @@ macro_rules! assert_query {
 }
 
 #[macro_export]
-macro_rules! assert_query_many {
-    ($runner:expr, $q:expr, $potential_results:expr) => {
+macro_rules! connector_results {
```

Comment:
I think I proposed this name initially when I wrote you, but that was more or less a placeholder as naming this is hard 😄 `diverging_results` maybe? `possible_results`? something along those lines perhaps.